packages/react-reconciler/src/ReactFiberBeginWork.js

```diff
@@ -1035,25 +1033,6 @@ function updateContextConsumer(current, workInProgress, renderExpirationTime) {
   return workInProgress.child;
 }
 
-/*
-  function reuseChildrenEffects(returnFiber : Fiber, firstChild : Fiber) {
-    let child = firstChild;
-    do {
-      // Ensure that the first and last effect of the parent corresponds
-      // to the children's first and last effect.
-      if (!returnFiber.firstEffect) {
-        returnFiber.firstEffect = child.firstEffect;
-      }
-      if (child.lastEffect) {
-        if (returnFiber.lastEffect) {
-          returnFiber.lastEffect.nextEffect = child.firstEffect;
-        }
-        returnFiber.lastEffect = child.lastEffect;
-      }
-    } while (child = child.sibling);
-  }
-  */
-
```

Comment:
This code is here for future reference.

Reply:
Thought! Can we not get the code from Git History for future reference?

Reply:
@gaeron add back the commented code for now.